uberduck_ml_dev/trainer/base.py

```diff
@@ -193,6 +201,27 @@ def log_training(
             scalar=step_duration_seconds,
         )
 
+        batch_levels = X[5]
+        batch_levels_unique = torch.unique(batch_levels)
+        for l in batch_levels_unique:
+            mlb = mel_loss_batch[torch.where(batch_levels == l)[0]].mean()
+            self.log(
+                "MelLoss/train/" + str(l),
```

Comment:
Here and in the two other cases below, could we do something like `f"MelLoss/train/speaker{l.item()}"` so that the graphs are labeled with names like `speaker0` instead of `tensor(0, device="cuda:0")`

Reply:
good point